packages/instant/src/components/time_counter.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+
+import { timeUtil } from '../util/time';
+
+import { Flex } from './ui/flex';
+import { Text } from './ui/text';
+
+export interface TimeCounterProps {
+    estimatedTimeMs: number;
+    ended: boolean;
```

Comment:
nit: `isEnded`

Reply:
Or `hasEnded`

Reply:
... or something

Reply:
ah!  the boolean prefix convention still hasn't been wired into my thinking yet :)

Reply:
it looks like there is an `eslint` rule for this, but not a `tslint` rule: https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/boolean-prop-naming.md 😢

Reply:
I'm a bit confused about tslint. We definitely(?) have this in website, and also a rule about adding `Async` at the end of async functions. Is that not working?